src/Microsoft.Azure.WebJobs.Host/Loggers/ValueWatcher.cs

```diff
@@ -15,120 +15,4 @@
 
 namespace Microsoft.Azure.WebJobs.Host.Executors
 {
-    internal class ValueWatcher
```

Comment:
I filed [https://github.com/Azure/azure-webjobs-sdk/issues/1217](https://github.com/Azure/azure-webjobs-sdk/issues/1217) to track down whether the references got deleted erroneously. 

---
In reply to: [126188615](https://github.com/Azure/azure-webjobs-sdk/pull/1209#discussion_r126188615) [](ancestors = 126188615,125713772)